frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -673,6 +674,7 @@ export class RotkehlchenApi {
         ...payload
       }),
       {
+        timeout: 60000,
```

Comment:
The default timeout of 30 seconds is enough, 60 seconds is too much. 
If the creation of an async takes more than 30 seconds the fix is not to increase the timeout on the frontend, we raise the issue with the backend people.